.changeset/long-sheep-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-graph': minor
+---
+
+Add the entire `Entity` to `EntityNodeData`
```

Comment:
Could you write a bit more details about what changed and add one example of how to use the entity instead of the individual properties?